zebrad/tests/common/get_block_template_rpcs/get_block_template.rs

```diff
@@ -27,9 +30,18 @@ use crate::common::{
 /// We've seen it take anywhere from 1-45 seconds for the mempool to have some transactions in it.
 pub const EXPECTED_MEMPOOL_TRANSACTION_TIME: Duration = Duration::from_secs(45);
 
+/// Delay between getting block proposal results and cancelling long poll requests.
+pub const EXTRA_LONGPOLL_WAIT_TIME: Duration = Duration::from_millis(150);
+
+/// Delay between attempts to validate a template as block proposals.
+///
+/// Running many iterations in short intervals tests that long poll requests correctly
+/// return `submit_old: false` when the old template becomes invalid.
+pub const BLOCK_PROPOSAL_INTERVAL: Duration = Duration::from_millis(300);
```

Comment:
This is exactly `DEFAULT_MAX_REQUEST_DELAY`, should it be set to that constant, so it automatically adapts if CI gets more unreliable? 
```suggestion
pub const BLOCK_PROPOSAL_INTERVAL: Duration = zebra_test::mock_service::DEFAULT_MAX_REQUEST_DELAY;
```

Reply:
I think I need to increase this interval, it should be unrelated to CI reliability because it's only meant to be a minimum that ensures the test runs across various chain states regardless of how long `try_validate_block_template` takes.